chart/values.yaml

```
@@ -1251,7 +1252,7 @@ dags:
     # Volume size for dags
     size: 1Gi
     # If using a custom storageClass, pass name here
-    storageClassName:
+    storageClassName: 
```

```suggestion
    storageClassName:
```

Unintentional change?